ofprotocol/ofp13/ofp13_parser_test.go

```diff
@@ -3972,7 +3973,7 @@ func TestSerializeMeterBandDscpExperimenter(t *testing.T) {
 /* OfpPacketIn                                       */
 /*****************************************************/
 func TestParsePacketIn(t *testing.T) {
-	packet := []byte{
+	packetWithoutPacketIn := []byte{
```

Comment:
I think this byte array represents "PacketIn" packet data, so  "packetWithoutPacketIn" is not suitable for this variable
I think that other names are preferable, for example "packetInWithoutReceivedData".
What do you think about ?